origins_workflow/js/moderation_sidebar_fix.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file
+ * Defines Javascript behaviors for moderation sidebar.
+ */
+(function ($, Drupal) {
+  Drupal.behaviors.landinglisting = {
+    attach: (context) => {
+      $("a.archive-link").click(function(event) {
+        /*
+        A class of 'archive-link' means that this link has been faked by code in
+        origins_workflow_moderation_sidebar_alter, so we complete the illusion
+        by adding a confirmation here in the same way that moderation sidebar
+        normally would.
+         */
+        return confirm("When archived, this content will be unpublished. Are you sure that you want to archive?");
```

Comment:
Worth adding a translation wrapper around this? If it's a stop-gap to tide us over in the event of a moderation sidebar bug then probably not but it caught my eye, naturally just as I approved the PR :)

Reply:
Yes, good point, have done that